pubtools/_quay/container_image_pusher.py

```diff
@@ -55,7 +55,8 @@ def quay_client(self):
             )
         return self._quay_client
 
-    def run_tag_images(self, source_ref, dest_refs, all_arch):
+    @classmethod
```

Comment:
Tag docker was implemented on top of another change which is currently in PR (#14 ). Could you only take into account commits starting from "Implement tag docker"? Thanks.

As for the comment itself, I prefer to keep it as a class method, since I can still use the self.<method> syntax. But if you insist then I can change it.